cev_eris.dme

```diff
@@ -3136,4 +3136,5 @@
 #include "zzzz_modular_occulus\prosthesis\implementation.dm"
 #include "zzzz_modular_occulus\prosthesis\prosthesis_brands.dm"
 #include "zzzz_modular_occulus\prosthesis\prosthesis_parts.dm"
+#include "zzzz_modular_occulus\icons\mob\more_hair.dmi"

```

Comment:
DMIs do not need to be included in .dmes, and even if they were, this is improperly sorted (dreammaker should automatically sort this on its own.)